model/base/src/main/java/org/eclipse/ditto/model/base/headers/DittoHeaderDefinition.java

```diff
@@ -230,7 +230,7 @@
      *
      * @since 1.1.0
      */
-    WWW_AUTHENTICATE("www-authenticate", String.class, false, true,
+    WWW_AUTHENTICATE("www-authenticate", String.class, true, true,
```

Comment:
I think it could result in strange behaviours when we read this from external headers. I also don't understand what the benefit could be.

Reply:
Yes, you're right .. I'll undo this one.